demisto_sdk/commands/unify/generic_module_unifier.py

```diff
@@ -0,0 +1,61 @@
+"""
+Unifies a GenericModule object with it's Dashboards
+"""
+import json
+import os
+from typing import Dict, Optional
+
+
+def find_dashboard_by_id(pack_path: str, dashboard_id: str) -> Optional[Dict]:
+    """
+    Search for dashboard with the given id in the given pack path.
+
+    Args:
+        pack_path: a pack to search in
+        dashboard_id: dashboard id to search for
+
+    Returns: if found - the content of the dashboard, else - None.
+    """
+    for file_name in os.listdir(pack_path):
+        file_path = os.path.join(pack_path, file_name)  # TODO: need to make sure how exactly the dashboards located in the pack
+        if file_path.endswith('.json') and file_name.startswith('dashboard'): # it's a dashboard
+            with open(file_path) as f:
+                dashboard = json.load(f)
+            if dashboard.get('id') == dashboard_id: # dashboard was found
+                return dashboard
+    return None
+
+
+def insert_dashboards_to_generic_module(generic_module_path: str) -> Dict:
+    """
+    Unifies a GenericModule object with it's Dashboards
+    Args:
+        generic_module_path: a path to a GenericModule file (json)
+
+    Returns: the unified GenericModule
+
+    """
+    pack_path = f'{os.path.dirname(os.path.dirname(generic_module_path))}/'
+    pack_name = os.path.basename(pack_path.rstrip('/'))
+
+    with open(generic_module_path) as f:
+        generic_module = json.load(f)
+
+    if views := generic_module.get('views'):
```

Comment:
SDK is written to support python 3.7 and up. I think lower python versions would throw an error about walrus syntax - same for the rest of the uses of this.